build_registry/project_registration.json5

```diff
@@ -7,84 +7,24 @@
 {
   projects: [
     {
-      project_id: "US_DOE_IEF_2023",
-      config_file: "project/project.json5",
-    },
+      project_id: "US_DOE_IEF_2025",
+      config_file: "project/project.json5"
+    }
   ],
   datasets: [

```

Comment:
Shouldn't this also list the buildings dataset if we're merging into main? Have you merged main back into this branch yet and confirmed that doing so keeps everything you need for buildings as well?